### ISSUE TITLE
Background and lineColor don't seem to be accepted 

### ISSUE THREAD

**Issue by alex-dotdev** [wonsikin/react-native-barcode-builder#11]
Packages:
```
"react": "16.0.0-alpha.6",
"react-native": "0.44.0",
"react-native-svg": "^5.1.8"
```

Have set the package without a problem and the barcode is being generated but the values for lineColor and background aren't taking effect.

```
import React, { Component } from 'react'
import { View, Dimensions } from 'react-native'
import Barcode from 'react-native-barcode-builder'

let width = Dimensions.get('window').width

export default class AppBarcode extends Component {
  constructor (props) {
    super(props)
  }
  render () {
    return (
      <View style={{
        flex: 1,
        marginHorizontal: 20,
        width: width * 0.8,
        marginVertical: 20,
        alignItems: 'center',
        flexDirection: 'column',
        justifyContent: 'center'}}>
        <Barcode value='4324324FVF' background='#ffffff' lineColor='#3fa2f7' text='None' format='CODE128'/>
      </View>
    )
  }
}
```

**Comment by wonsikin** [comment]
@alex-dotdev Hi, I fixed it, use the latest v1.0.1 version
![83b86fd8-0f0b-4d2c-9d79-54f7d2b9eaa1](https://cloud.githubusercontent.com/assets/6659426/26618501/2aca8828-460d-11e7-8040-5bc0324f342c.png)

**Comment by alex-dotdev** [comment]
Tested v1.0.1 and all working!
Thank you
